spec/save_inventory/spec_helper.rb

```diff
@@ -1,7 +1,7 @@
 module SpecHelper
   def assert_all_records_match_hashes(model_classes, *expected_match)
     # Helper for matching attributes of the model's records to an Array of hashes
-    model_classes = model_classes.kind_of?(Array) ? model_classes : [model_classes]
+    model_classes = [model_classes] unless model_classes.kind_of?(Array)
```

Comment:
Think this is the same thing:

```suggestion
    model_classes = Array.wrap(model_classes)
```